test/basic.html

```diff
@@ -168,6 +168,28 @@
 						{value: 5, label: 'ji'}
 					];
 				});
+
+				test('Search with special characters', done => {
+					chips.textProperty = 'label';
+					autocomplete.text = '-class & A.B selection(f';
+
+					window.setTimeout(() => {
+						assert.equal(suggestions._suggestions.length, 1);
+						assert.equal(suggestions._suggestions[0].value.label, 'Q-class & A.B selection(fake)');
+						done();
+					}, 200);
```

Comment:
Maybe move this block to the end of the test, so all code is in the order it is ran.